types/requests.ts

```diff
@@ -245,3 +245,78 @@ export interface CodeExecutionTool {
    */
   codeExecution: {};
 }
+
+/**
+ * Request message for generating image.
+ * @public
+ */
+export interface ImageGenerationRequest {
+  /**
+   * Text prompt for the image.
+   */
+  prompt: string;
+  /**
+   * A description of what you want to omit in the generated images.
+   */
+  negativePrompt?: string;
+  /**
+   * Number of images to generate. Range: 1..8.
+   */
+  numberOfImages?: number;
+  /**
+   * Width of the image. One of the Width/Height sizes must be 256 or 1024.
+   */
+  width?: number;
+  /**
+   * Height of the image. One of the Width/Height sizes must be 256 or 1024.
+   */
+  height?: number;
```

Comment:
Do we provide any validation for these values? What kind of error will the user see if they provide an invalid value (e.g. numerOfImages = 10)?

Reply:
The value validation would be done at the backend server. If they provide the wrong argument, the response status would be INVALID_ARGUMENT.